### PR TITLE
[FEATURE] StealCommand

### PR DESCRIPTION
**Pull request by Lolicon2110**
Currently untested

### CODE REVIEW

File: src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  @Override
+  protected void execute(MessageReceivedEvent e, String[] args) {
+    MessageChannel channel = e.getChannel();
+    String name = args[1];
+    if (name.length() <= 32 && name.length() >= 2) {
+      if (args.length == 2) {
+        try {
+          URL url = new URL(args[0]);
```
Comment (d1snin) [inline src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java:30]: arguments to the command start with args[1]
 so `;command` - args[0]

**Review by d1snin — CHANGES_REQUESTED** [review]

File: src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java (hunk 1 above)
Comment (d1snin) [reply src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java:30]: also u need to use this
`URL url = new URL(e.getMessage().getEmotes().get(0).getImageUrl());` instead

**Comment by d1snin** [comment]
I would also correct here on my own, so if you want, I will change it as it should and merge

File: src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java (hunk 1 above)
Comment (Lolicon2110) [reply src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java:30]: alright i'll fix the first point, about the second point, i was aiming to have this be usable by people without nitro, that being the reason for a raw url, should i add something to check if there is an emote in the message and do what you said if it exists, or just make it only what you said?

Comment (Lolicon2110) [reply src/main/java/xyz/d1snin/corby/commands/misc/StealCommand.java]: first point fixed, waiting for input on my last comment

**Review by d1snin — APPROVED** [review]

**Comment by d1snin** [comment]
ok i rewrited it a "bit"

**Review by d1snin — APPROVED** [review]

**Comment by d1snin** [comment]
thanks for your pr!

**Pull request merged**